lib/bigquery/dataset.js

```diff
@@ -56,6 +56,9 @@ function Dataset(bigQuery, id) {
 /**
  * Create a table given a tableId or configuration object.
  *
+ * See [Tables: insert](https://cloud.google.com/bigquery/docs/reference/v2/tables/insert)
```

Comment:
So initially I started to go that route, but I think if we were to upgrade the api version it would be pretty annoying to regenerate all the short urls. I think by including the full path, it'll be easier to update in the future - e.g. `v1` => `v2'